app/assets/javascripts/SourceCodeGlower/AnnotationTextDisplayer.js

```diff
@@ -72,6 +72,7 @@ AnnotationTextDisplayer.prototype.hide = function() {
 // Show the displayer
 AnnotationTextDisplayer.prototype.show = function() {
   this.display_node.style.display = 'block';
+  reloadDOM();
```

Comment:
Because it's defined it a preceding file, which happens to share the same scope.